Wikipedia/Code/WMFSettingsViewController.m

```diff
@@ -89,6 +89,7 @@ - (void)viewDidAppear:(BOOL)animated {
 
 - (void)viewWillAppear:(BOOL)animated {
     [super viewWillAppear:animated];
+    self.navigationController.toolbarHidden = YES;
```

Comment:
I had formerly set the nav controller's toolbar to hidden in `WMFHelpViewController`'s `viewWillDisappear` but for some reason after tapping a few links and then tapping the "W" at the top to return to the settings screen *sometimes* the toolbar would still be visible when the settings screen appeared...